docs/examples/notebooks/audio_decoder.ipynb

```diff
@@ -0,0 +1,235 @@
+    "else:\n",
+    "    print(f\"ROCAL_DATA_PATH is set to: {rocal_data_path}\")\n",
+    "\n",
+    "rocal_audio_data_path = os.path.join(rocal_data_path, \"rocal_data\", \"audio\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2cb7c80e-fa78-4fdb-993e-f1f0eb4d62ba",
+   "metadata": {},
+   "source": [
+    "Configuration of the rocAL Pipeline:\n",
+    "\n",
+    "The rocAL pipeline is configured with the following parameters:\n",
+    "\n",
+    "    batch_size: 1\n",
+    "    CPU/GPU Backend: Configured to use CPU (rocal_cpu=True)\n",
+    "\n",
+    "Using a batch_size of 1 and cpu backend to keep things simple"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3a4d4365-5f01-4923-a0e4-6c13abb52883",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file_list = f\"{rocal_audio_data_path}/wav_file_list.txt\" #Use file list defined in the MIVisisonX-data repo\n",
+    "rocal_cpu = True\n",
+    "audio_pipeline = Pipeline(batch_size=1, num_threads=8, rocal_cpu=rocal_cpu)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d59b9f7a-c997-4155-a9dd-81adf9c31f81",
+   "metadata": {},
+   "source": [
+    "Reading Audio and Labels: \n",
+    "\n",
+    "    The 'fn.readers.file' function reads audio files and their labels from the provided file list.\n",
+    "\n",
+    "Decoding Audio:\n",
+    "\n",
+    "    The fn.decoders.audio function decodes the audio data with specified parameters.\n",
+    "\n",
+    "Parameters used for decoding:\n",
+    "\n",
+    "    audio: The audio data to be decoded.\n",
+    "    file_root: The base path where audio wav files are present.\n",
+    "    file_list_path: The path to the file list of audio wav files.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2e391975-3769-41ac-9a52-28872b6a81c4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with audio_pipeline:\n",
+    "    audio, labels = fn.readers.file(file_root=rocal_audio_data_path, file_list=file_list)\n",
+    "    decoded_audio = fn.decoders.audio(\n",
+    "        audio,\n",
+    "        file_root=rocal_audio_data_path,\n",
+    "        file_list_path=file_list)\n",
+    "    audio_pipeline.set_outputs(decoded_audio)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a8fbfade",
+   "metadata": {},
+   "source": [
+    "Build the pipeline and pass the pipeline to the ROCALAudioIterator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b8e80c14-7521-4489-b3fd-7c300e2c144a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "audio_pipeline.build()\n",
+    "audioIterator = ROCALAudioIterator(audio_pipeline)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1bb26e37-3d99-4da4-92f6-004cb4de8c6c",
+   "metadata": {},
+   "source": [
+    "The output from the iterator includes the audio data as PyTorch tensors, the corresponding labels, and the region of interest."
```

Comment:
please add the display for the ouput of the pipeline as it is standard in notebooks to show the final output